test/sqllogictest/transform/threshold_elision.slt

```diff
@@ -431,24 +431,24 @@ WITH MUTUALLY RECURSIVE
 SELECT * FROM c1
 ----
 Explained Query:
-  Return // { non_negative: true }
-    Get l0 // { non_negative: true }
+  Return // { non_negative: false }
```

Comment:
Regression in non-negativity (which tracks until we update its pattern detector on unions).